package.json

```diff
@@ -32,6 +32,7 @@
     "test-unit-android": "node ./scripts/test-unit --android",
     "pretest-e2e-android": "npm run build",
     "test-e2e-android": "node ./scripts/test-e2e --android",
+    "test-e2e-android-locked": "node ./scripts/test-e2e --android --locked",
```

Comment:
At its current state, this PR requires a dedicated emulator and for the user to manually setup lock screen with `1234` as password.
Until we can automate this (Probably with UiAutomator) I'd like to remove it from `test-e2e.js` script and into a dedicated script.